MarkdownLivePreview.py

```diff
@@ -78,11 +141,8 @@ def run(self, edit):
         )
 
         preview_window.focus_group(1)
-        preview_view = preview_window.new_file()
-        preview_view.set_scratch(True)
-        preview_view.settings().set(PREVIEW_VIEW_INFOS, {})
-        preview_view.set_name("Preview")
-        # FIXME: hide number lines on preview
+
+        preview_view = mdpopups.new_html_sheet(preview_window, "Preview", "")
```

Comment:
I suggest changing this to:

`preview_view = mdpopups.new_html_sheet(window=preview_window, name="Preview", contents="")`